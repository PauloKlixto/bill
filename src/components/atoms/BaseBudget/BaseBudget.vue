<template>
  <section class="BaseBudget">
    <p>{{ value }} left</p>
  </section>
</template>

<script>
export default {
  name: 'BaseBudget',
  props: {
    value: {
      type: [String, Number],
      default: ''
    }
  }
}
</script>

<style lang="scss" scoped>
.BaseBudget {
  background-color: $color-green;
  max-width: 950px;
  width: 100vw;
  margin: auto;
  color: $color-white;
  text-align: center;
  font-weight: bold;
  font-size: 32px;
  padding: 30px 0;
  position: sticky;
  top: 0;
  z-index: 1;
}
</style>
