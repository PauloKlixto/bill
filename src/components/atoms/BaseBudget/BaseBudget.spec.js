/* eslint-disable no-undef */
import { shallowMount } from '@vue/test-utils'
import BaseBudget from './BaseBudget'

describe('BaseBudget', () => {
  const wrapper = shallowMount(BaseBudget)

  test('Componente existe', () => {
    expect(wrapper.exists()).toBe(true)
  })

  test('Componente nÃ£o alterado', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
