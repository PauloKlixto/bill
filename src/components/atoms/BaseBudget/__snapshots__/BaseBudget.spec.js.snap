// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BaseBudget Componente n√£o alterado 1`] = `
"<section class=\\"BaseBudget\\">
  <p> left</p>
</section>"
`;
